.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_*
+*.log
+logs
+**/*.backup.*
+**/*.back.*
+
+node_modules
+bower_components
```

Comment:
I think this was added by the starter kit. Right?

Reply:
From https://gitignore.io/

Reply:
Yes. We won't be using bower ;)